src/index.ts

```diff
@@ -1,4 +1,4 @@
-const secp256k1 = require('secp256k1')
+const secp256k1 = require('ethereum-cryptography/secp256k1')
```

Comment:
This variable gets exported in the line 20, so it's a breaking change. 

We need a wrapper of secp256k1 v4 that offers the same API than secp256k1 v3. 

The same happens in the version 5 of this library. Luckily, v7 doesn't reexport secp256k1 anymore.

Reply:
I think @nebojsa94 is working on this. Can you confirm?

Reply:
Yep